design/lvm/resize_workflow.md

```diff
@@ -139,7 +139,9 @@ A test plan will include following test cases:
 - Test volume expansion on statefulset application with multiple replicas.
 - Test volume expansion of thin provisioned volume.
 - Test volume expansion with snapshot(s).
-- Test volume expansion for thick provisioned volume group by increasing size greater then vg size.
+- Test volume expansion of thick provisioned volume group by increasing volume size greater than underlying vg size.
```

Comment:
let us add what to verify in the test case.

Reply:
Is it something like below?
```
- Test volume expansion with snapshot(s) and verify volume should remain in same size
- Test volume expansion of thick provisioned volume by increasing volume size greater than underlying vg size and verify accessibility of volume
```

Reply:
yes